src/eosio-config/config.ini

```diff
@@ -344,7 +344,7 @@ state-history-endpoint = 0.0.0.0:8080
 # txn-test-gen-account-prefix = txn.test.
 
 # Plugin(s) to enable, may be specified multiple times
-plugin = eosio::history_api_plugin
+# plugin = eosio::history_api_plugin
```

Comment:
Why is this removed? Is this no longer working or not needed?

Reply:
It is no longer present in the more recent versions of nodeos